tests/IotServiceTest.cpp

```diff
@@ -74,17 +74,16 @@ static int s_TestIotPublishSubscribe(Aws::Crt::Allocator *allocator, void *ctx)
         bool subscribed = false;
         bool published = false;
         bool received = false;
-        auto onConnectionCompleted =
-            [&](MqttConnection &, int errorCode, ReturnCode returnCode, bool sessionPresent) {
-                printf(
-                    "%s errorCode=%d returnCode=%d sessionPresent=%d\n",
-                    (errorCode == 0) ? "CONNECTED" : "COMPLETED",
-                    errorCode,
-                    (int)returnCode,
-                    (int)sessionPresent);
-                connected = true;
-                cv.notify_one();
-            };
+        auto onConnectionCompleted = [&](MqttConnection &, int errorCode, ReturnCode returnCode, bool sessionPresent) {
```

Comment:
what's causing these changes?  clang-format is not making this change for me

Reply:
Clang-format did this for me though.